code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -110,6 +110,13 @@
 	ammo_type = /obj/item/ammo_casing/shotgun/tranquilizer
 	materials = list(MAT_METAL=1750)
 
+/obj/item/ammo_box/tranqdarts

```

Comment:
If it would help, since these aren't shotgun shells but rather their own thing, I can change them from being a 20g style shell and make them into a normal copper shell casing.